### ISSUE TITLE
Interacting with chests causes small-stack items to stack to 64

### ISSUE THREAD

**Issue by RS-GoogleCoder** [Fureniku/crafting-station#4]
In the image below is a chest with 4 stacks of signs. (Signs only stack to 16.) From the chest GUI, I cannot combine these stacks.
![image](https://github.com/user-attachments/assets/1c82d117-12d2-4786-b9d4-99ec6724afa7)

But when I open the crafting station GUI (with the chest on the left)
![image](https://github.com/user-attachments/assets/49cec342-9d78-4f6b-b3c4-4489ac31e58d)

I am able to combine the signs into a stack of 64
![image](https://github.com/user-attachments/assets/597dd815-a04c-4b5e-8a04-0d920e0a5c35)

I am using a modpack: Gregtech Community Pack Modern

**Comment by Fureniku** [comment]
I actually saw the code that's doing this yesterday and I'm not sure why it was ever added, but I figure there must be some reason and didn't want to touch it for now. Just testing it seems to be mostly a visual issue (if you go back to the chest and pick up the signs, it'll only let you put down 16s at once still), but yes does let you bypass the limit for the specific inventory temporarily.

I've removed the code in my local branch now but I'm going to do more testing to make sure its not breaking some edge case. Original creators must've had some reason to add it...
